src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -330,7 +327,7 @@ namespace winrt::TerminalApp::implementation
                 }
                 else
                 {
-                    return;
+                    co_return;
```

Comment:
Michael made this same change in his DefTerm implementation so I'm not sure why it appears as an issue sometimes. Either way, I think this is more correct because we're `co_await`-ing right before it. So the idea is you're supposed to `co_return` on a `co_await` or `fire_and_forget`.